pkg/kubecost/allocation_test.go

```diff
@@ -1751,6 +1778,12 @@ func TestAllocationSet_ReconcileAllocations(t *testing.T) {
 	for key := range as.idleKeys {
 		as.Delete(key)
 	}
+	// add reconcilable pvs to pod-mno
+	for _, a := range as.allocations {
+		if a.Properties.Pod == "pod-mno" {
+			a.AddPVBreakDown(map[string]PVUsage{"disk1": {Cost: 2.5, ByteHours: 2.5 * gb}, "disk2": {Cost: 5, ByteHours: 5 * gb}})
```

Comment:
This is a pretty brutal line. Maybe we can multi-line and indent it for readability?